### ISSUE TITLE
1541 Disk directory and 1541 Disk contents unsupported for .g64 images?

### ISSUE THREAD

**Issue by c64cryptoboy** [slajerek/RetroDebugger#52]
**Describe the bug**
1541 Disk directory and 1541 Disk contents views seem to work with .d64 images, but not .g64 images.

**To Reproduce**
1. File->Open a .g64 image
2. open C64->1541 Drive->1541 Disk directory
3. open C64->1541 Drive->1541 Disk contents

**Expected behavior**
To function as they do with d64 files.

**Screenshots**
Behavior with a .d64 file:
![image](https://github.com/slajerek/RetroDebugger/assets/21978243/ded503b5-532b-4ce6-be27-297ad1b8a9c7)

Behavior with a .g64 file:
![image](https://github.com/slajerek/RetroDebugger/assets/21978243/ee5dac5b-be86-4d7c-8069-0753df8b3b70)

**Desktop (please complete the following information):**
 - OS: Windows 11 Home 23H2
 - Retro Debugger v0.64.68

**Comment by slajerek** [comment]
Yes, that is correct. Support for *.g64 is still in progress indeed.
